deploy/helms/legion-core/templates/jenkins/jenkins-pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.jenkins.enabled }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: "{{ .Release.Name }}-jenkins-master-home"
+  labels:
+    app: "{{ .Release.Name }}-jenkins-master-home"
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  accessModes:
+    - {{ .Values.jenkins.persistence.accessMode | quote }}
+  resources:
+    requests:
+      storage: {{ .Values.jenkins.persistence.size | quote }}
+{{- if .Values.jenkins.persistence.storageClass }}
+{{- if (eq "-" .Values.jenkins.persistence.storageClass) }}
```

Comment:
Why do we need '-' as a default storage class?

Reply:
We use next logic:
- "-" for zero string (default) class
- "" for skipping storageClass declaration
- any other string for specific class.

Reply:
`PVCs don't necessarily have to request a class. A PVC with its storageClassName set equal to "" is always interpreted to be requesting a PV with no class, so it can only be bound to PVs with no class (no annotation or one set equal to ""). A PVC with no storageClassName is not quite the same and is treated differently by the cluster depending on whether the DefaultStorageClass admission plugin is turned on.`

https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1

Reply:
@akharlamov  So, we cannot remove that parameter, because in our cluster we use default PVC storage class that has non empty name